build.gradle.kts

```diff
@@ -134,7 +134,7 @@ tasks {
     register<Exec>("addWindowsAdminUser") {
         workingDir("keycloak/keycloak-$keycloakVersion/bin")
         commandLine("cmd", "/c", "add-user-keycloak.bat", "-r", "master", "-u", "admin", "-p", "admin")
-        environment("JAVA_OPTS", "--add-modules=java.se")
+        //environment("JAVA_OPTS", "--add-modules=java.se")
```

Comment:
if its not necessary you might just remove it :)

Reply:
removed it